### PR TITLE
Replaced selectByValue with selectByAttribute

### PR DESCRIPTION
**Pull request by Risto-Stevcev**
Replacing selectByValue with selectByAttribute (see #1082)

### CODE REVIEW

**Comment by christian-bromann** [comment]
@Risto-Stevcev nice. Could you also remove the logic in `selectByValue` and use `selectByValue` there?

**Comment by Risto-Stevcev** [comment]
@christian-bromann What do you mean? keep `selectByValue` but have it run `selectByAttribute` under the hood?

**Comment by christian-bromann** [comment]
> keep selectByValue but have it run selectByAttribute under the hood?

Yes, exactly!

**Comment by Risto-Stevcev** [comment]
Sure! like this?

File: lib/commands/selectByValue.js [hunk 1]
```diff
@@ -34,38 +34,10 @@
...
-         */
-        var normalized = '[normalize-space(@value) = "' + value.trim() + '"]'
-        return this.elementIdElement(res.value.ELEMENT, `./option${normalized}|./optgroup/option${normalized}`)
-    }).then((res) => {
-        /**
-         * select option
-         */
-        return this.elementIdClick(res.value.ELEMENT)
-    })
+  return selectByAttribute.call(this, selectElem, 'value', value)
```
Comment (christian-bromann) [inline lib/commands/selectByValue.js]: just use `return this.selectByAttribute(...)`

Comment (Risto-Stevcev) [inline lib/commands/selectByValue.js]: the original function uses `this.element` and complains if it doesn't rebind `this`.

Comment (christian-bromann) [inline lib/commands/selectByValue.js]: if you use `return this.selectByAttribute(...)` you automatically bind this

Comment (Risto-Stevcev) [inline lib/commands/selectByValue.js]: Oh I see what you're saying, didn't see the `this.` part. Ok, no problem.

**Comment by christian-bromann** [comment]
Sweet, LGTM!

**Pull request merged**